paths.pegjs

```diff
@@ -62,7 +62,7 @@ SimpleExpression = head:WordOrParen tail:(Separator WordOrParen)+ {
   }
 } / EquationPart
  
-DateOperation = operation:("date"/"datetime"/"addmonths"/"addyears"/"adddays"/"addweeks"/"addhours") _ "(" _ path:SimpleExpression secondParameter:(PathSeparator SimpleExpression)? _ ")" {
+DateOperation = operation:("datetime"/"date"/"addmonths"/"addyears"/"adddays"/"addweeks"/"addhours") _ "(" _ path:SimpleExpression secondParameter:(PathSeparator SimpleExpression)? _ ")" {
```

Comment:
for some reason the order here really mattered ---- it hit "date" and moved on and didn't have a way to back out?

i thought peg would be better about that, but it wasn't.

Reply:
Yeah peg just takes the first match so that makes sense